final class UnknownError implements Exception {
  const UnknownError();
}
